src/features/customerView/leasesCard/BerthLeasesCard.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { BerthLease } from '../types';
+import LeasesCard from './LeasesCard';
+
+export interface BerthLeasesCardProps {
+  leases: BerthLease[];
+  handleShowContract(leaseId: string): void;
+}
+
+const BerthLeasesCard = ({ leases, handleShowContract }: BerthLeasesCardProps) => {
+  const { t } = useTranslation();
+  const leaseDetails = leases.map((lease) => {
+    return {
+      id: lease.id,
+      endDate: lease.endDate,
+      startDate: lease.startDate,
+      link: lease.harbor ? `/harbors/${lease.harbor.id}` : undefined,
+      address: `${lease.harbor?.name || ''} ${lease.pierIdentifier || ''} ${lease.berthNum}`,
```

Comment:
To eliminate empty address parts, this could be replaced as follows (used for address formatting in `format.ts`):
```[lease.harbor?.name || '', lease.pierIdentifier || '', lease.berthNum].filter(Boolean).join(' ')```

Reply:
Done.